app/utils/IncomeSourcesUtils.scala

```diff
@@ -18,13 +18,19 @@ package utils
 
 import auth.MtdItUser
 import config.featureswitch.{FeatureSwitching, IncomeSources}
+import enums.IncomeSourceJourney.IncomeSourceType
+import enums.JourneyType.{Add, JourneyType}
+import models.incomeSourceDetails.AddIncomeSourceData
 import play.api.mvc.Result
 import play.api.mvc.Results.Redirect
+import services.SessionService
 import uk.gov.hmrc.auth.core.AffinityGroup.Agent
+import uk.gov.hmrc.http.HeaderCarrier
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
 
 trait IncomeSourcesUtils extends FeatureSwitching {
+
```

Comment:
* IncomeSourceUtils trait will be remove when this part of domain will be finally activated / enabled in prod, no need to add any extra logic within it, you can create a new trait ie. like trait JourneyChecker or JourneyManager and extend it with IncomeSourcesUtils trait